include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -868,15 +868,52 @@ namespace internal
       // copy constrain matrix; TODO: why only for the coarse level?
       transfer.constraint_coarse.copy_from(constraint_coarse);
 
-      // make sure that no hp is used
-      AssertDimension(dof_handler_coarse.get_fe_collection().size(), 1);
-      AssertDimension(dof_handler_fine.get_fe_collection().size(), 1);
+      // gather ranges for active FE indices on both fine and coarse dofhandlers
+      std::array<unsigned int, 2> min_active_fe_indicices = {
```

Comment:
```suggestion
      std::array<unsigned int, 2> min_active_fe_indices = {
```
(and the same for `max_fe_indices`).

Reply:
Fixed.